packages/field-plugin/src/createFieldPlugin/createFieldPlugin.ts

```diff
@@ -18,7 +19,9 @@ export const createFieldPlugin: CreateFieldPlugin = (onUpdateState) => {
   if (!isEmbedded) {
     onUpdateState({
       type: 'error',
-      error: new Error(`The window is not embedded within another window`),
+      error: new Error(
+        `The window is not embedded within another window. Did you mean to open the field plugin in the sandbox? ${sandboxUrl()}`,
+      ),
```

Comment:
```suggestion
      error: new Error(
        process.env.NODE_ENV === 'development' ? `The window is not embedded within another window. Did you mean to open the field plugin in the sandbox? ${sandboxUrl()}` : `The window is not embedded within another window`,
      ),
```

Sorry I should've just written this code suggestion. I phrased it poorly. Does this make sense?

Reply:
@johannes-lindgren I'm not sure if I'm following here. We currently don't allow field-plugins deployed on their own url. So I cannot imagine how a field plugin can be opened outside Field Plugin Editor or Visual Editor in production.

localhost:xxxx can be opened directly, but then it's a development mode. So that's why I thought this error message is relevant to development mode only. Am I missing something?

Reply:
Ah... gotcha! Thanks for the explanation. I didn't think of that case.

Reply:
Yeah it's the first time we're able to deploy plugin preview like this 😀

Previously if we wanted to have a preview, we'd create and deploy a new plugin with a `-preview` suffix, like this: https://app.storyblok.com/#/partner/fields/41034

But this has a few drawbacks:

- They will need to be manually deployed. We could set up CI that for example deploys the `develop` branch to the `-preview` plugin, but it would be difficult to handle all the feature branches.
- It is not clear which branch is being previewed.
- We will not receive a URL in the PR.
- We cannot share the plugin easily. With Vercel, we can just send a link to the design team, partner manager, or whoever else.

😃